### PR TITLE
Added Readme

### PR DESCRIPTION
**Pull request by malscent**

### CODE REVIEW

**Review by patrick-stephens — COMMENTED** [review]
I think some of the sentences need splitting to new lines, it makes it harder to comment on them.

There's also a mix of formatting with 'text' or `text` that needs sorting I think.

**Comment by malscent** [comment]
Fixed a bunch of the issues you brought up @patrick-stephens let me know if you see anything else

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+This repo manages the CouchbaseServerAndSyncGateway Azure Application offering.  When the azure release is created, it will add two zip files to the artifacts:  `azure-cbs-archive-byol.zip` 
+and `azure-cbs-archive-hourly.zip`.  These two zip files are uploaded in the Azure Partner Center's Offer -> Plans. For azure, there is one offer with multiple "plans".  
+Each of these zip files correllates to the respective plan (byol or hourly)
+
+### **AWS Releases**
+
+When a AWS release is created, 4 Cloud Formation templates are added to the release's artifacts:  `aws-cbs-byol.template`, `aws-cbs-hourly-pricing.template`, `aws-sg-byol.template`,
+`aws-sg-hourly-pricing.template`.
+
+To update the public offering, you must take the "Product Load Form" named CouchbaseServerAndSyncGateway.xlsx in the root of the `./aws/CouchbaseServerAndSyncGateway/` folder.
```
Comment (patrick-stephens) [inline readme.md:36]: Back tick the xlsx?

**Review by patrick-stephens — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (malscent) [reply readme.md]: Yup.  and resolved!

**Pull request merged**